src/java.desktop/unix/native/libawt_xawt/awt/gtk3_interface.c

```diff
@@ -2861,35 +2863,48 @@ static void transform_detail_string (const gchar *detail,
     }
 }
 
+inline static int scale_down_to_plus_inf(int what, int scale) {
+    return (int)ceilf(what / (float)scale);
```

Comment:
I find the name of this and the companion function confusing.
what is "inf" short for ? I see that and I think plus_infinity.
scaled_ceiling and scaled_floor might be better names based on what I see.

Reply:
Right, "inf" is for infinity; this is sort of how the rounding modes are described in the IEEE754 standard. But since full length words make the function name too long and short versions are confusing, I will rename as you suggested.

Reply:
@prrace The change has been made, please, take a look.